internal/web/server.go

```diff
@@ -1122,3 +1123,36 @@ func isPortConflictError(err error) bool {
 	}
 	return false
 }
+
+func (s *server) repositoryconfig(w http.ResponseWriter, r *http.Request) {
+	switch r.Method {
+	case http.MethodGet:
+		// Handle GET request: Render the repository.html template
+		s.handleGetRepositoryConfig(w, r)
+	case http.MethodPost:
+		// Handle POST request: Update the repository configuration
+		// s.handlePostRepositoryConfig(w, r)
+		fmt.Printf("POST Mehtod Part")
+	default:
+		// Method not allowed
+		http.Error(w, "Method not allowed", http.StatusMethodNotAllowed)
+	}
+}
+
+func (s *server) handleGetRepositoryConfig(w http.ResponseWriter, r *http.Request) {
+	reposName = mux.Vars(r)["repoName"]
+	repos := v1alpha1.PackageRepositoryList{}
```

Comment:
this will now only hold one repository instead of a list, so please change it to a `v1alpha1.PackageRepository`